staging/src/k8s.io/kubectl/pkg/metricsutil/metrics_printer.go

```diff
@@ -295,7 +295,7 @@ func printAllResourceUsages(out io.Writer, metrics *ResourceMetricsInfo) {
 		fmt.Fprint(out, "\t")
 		if available, found := metrics.Available[res]; found {
 			fraction := float64(quantity.MilliValue()) / float64(available.MilliValue()) * 100
-			fmt.Fprintf(out, "%d%%\t", int64(fraction))
+			fmt.Fprintf(out, "%f%%\t", fraction)
```

Comment:
not binding, but I suggest to round it to 2 decimal places `"%.2f"`

Reply:
updated, 

now it will be something like 
```
NAME      CPU(cores)   CPU%     MEMORY(bytes)   MEMORY%
a010735   6558m        81.97%   10600Mi         67.16%
```